.github/workflows/main.yml

```diff
@@ -0,0 +1,23 @@
+name: Build static ios xcframework
+on: workflow_dispatch
```

Comment:
can you please add push and PRs as well?

Reply:
Added pull request as discussed